tests/unit/components/sl-chart-test.js

```diff
@@ -0,0 +1,363 @@
+    });
+
+    assert.strictEqual(
+        this.$().hasClass( 'sl-loading' ),
+        false,
+        'Default rendered component does not have "sl-loading" class'
+    );
+
+    Ember.run( () => {
+        component.set( 'isLoading', true );
+    });
+
+    assert.ok(
+        this.$().hasClass( 'sl-loading' ),
+        'Rendered component gains "sl-loading" class'
+    );
+});
+
+test( 'updateData() is called once', function( assert ) {
+    let component = this.subject({
+        options: testOptions, 
+        series: testSeries 
+    });
+
+    let spy = sinon.spy( component, 'updateData');
+
+    this.render();
+
+    assert.equal(
+        spy.calledOnce,
+        true,
+        'updateData() is called once'
+    );
+});
+
+test( '"Options" property needs to be an object', function( assert ) {
+    let assertionThrown = false;
+
+    try {
+        this.subject({
+            options: null,
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testOptions was empty'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: "string",
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testOptions was string'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: undefined,
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testOptions was undefined'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: true,
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testOptions was a boolean'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: 666,
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testOptions was number'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: function(){},
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testOptions was a function'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        !assertionThrown,
+        'testOptions was an Object'
+    );
+});
+
+test( '"Series" property needs to be an array', function( assert ) {
```

Comment:
There is a an extra space between the `}  catch(` in each of the blocks below that should be removed.